src/app/digital/internal/views/components/SwitchAssembler.ts

```diff
@@ -0,0 +1,107 @@
+import {SVGDrawing} from "svg2canvas";
+
+import {V}         from "Vector";
+import {Transform} from "math/Transform";
+
+import {Schema} from "core/schema";
+
+import {CircuitInternal}   from "core/internal";
+import {SelectionsManager} from "core/internal/impl/SelectionsManager";
+import {CircuitView}       from "core/internal/view/CircuitView";
+import {PortAssembler}     from "core/internal/view/PortAssembler";
+import {SVGPrim}           from "core/internal/view/rendering/prims/SVG";
+import {Style}             from "core/internal/view/rendering/Style";
+import {Assembler}         from "core/internal/view/Assembler";
+
+import {Signal}     from "digital/internal/sim/Signal";
+import {DigitalSim} from "digital/internal/sim/DigitalSim";
+import {Rectangle}  from "core/internal/view/rendering/prims/Rectangle";
+
+
+export class SwitchAssembler extends Assembler<Schema.Component> {
+    public readonly size = V(1.24, 1.54);
+    public readonly pressableSize = V(0.96, 1.2);
```

Comment:
Magic numbers, I don't want to have to figure out what these mean if/when I implement that eslint rule at some point in the future

Reply:
The magic numbers will have to exist somewhere, this is where - I'm trying to stray away from a monolithic `Constants.ts` file

I can make them static I guess

Reply:
What about having the constants defined in this file instead of a monolithic Constants.ts file? Just something like `const SWITCH_HORIZONTAL_SIZE = 1.24` or something like that? Or is that what you mean by making them static?

Reply:
An example of bad magic numbers is:
```ts
        if (transformChanged) {
            // Update transform
            this.view.componentTransforms.set(sw.id, new Transform(
                V(sw.props.x ?? 0, sw.props.y ?? 0),
                V(1.24, 1.54), <-------------
                (sw.props.angle ?? 0),
            ));
        }
```
These are named and their meaning is plenty obvious, putting them in a constant doesn't add anything here